Cakefile

```diff
@@ -409,6 +409,17 @@ runTests = (CoffeeScript) ->
         description: description if description?
         source: fn.toString() if fn.toString?
 
+  global.supportsAsync = if global.testingBrowser
```

Comment:
Do we really need this split? Could we not just always feature test it?

Reply:
Unfortunately not. See the comment at the top of `test/async.coffee`. Even with a feature test at the top of that file, and a `return` if it fails, a runtime that doesn't support async will throw an error just from trying to parse the file. Also there's an async test in `repl.coffee` now, so I want to feature test this once and cache the result rather than having it in two places.